test/unit/io.js

```diff
@@ -861,6 +861,54 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression does not throw error', function () {
```

Comment:
> I don't want to simply test that the file got smaller, since occasionally that doesn't work

Ah, that's along the lines of what I was thinking. Feel free to add a test fixture that can be relied upon to do this. I guess we can at least verify that the TIFF file(s) can be re-opened.